source/MaterialXTest/MaterialXRenderOsl/RenderOsl.cpp

```diff
@@ -175,6 +177,14 @@ bool OslShaderRenderTester::runRenderer(const std::string& shaderName,
             }
             CHECK(shader->getSourceCode().length() > 0);
 
+            // Convert relative paths to absolute, using hardcoded logic for now.
```

Comment:
This feels a bit too hard-coded. I added a flattenFilenames() utility a while back, but
this is an in-place Document modification, which is too much here. Still feels better to make this generic to resolve these paths vs string substitution more generally ?

Reply:
Agreed that this solution is not great, but it's the only approach I found that generates correct textured renders in OSL on WIndows.  There's definitely room for improvement here, but this at least gets us working renders across platforms.